src/main/java/com/exasol/exaconf/ConfigurationParser.java

```diff
@@ -51,10 +51,12 @@ private void parseUnindentedLine(final String line) {
             this.subsection = "";
         } else {
             final int assignmentOperatorPosition = line.indexOf('=');
-            final String key = line.substring(0, assignmentOperatorPosition - 1).trim();
-            final String value = line.substring(assignmentOperatorPosition + 1).trim();
-            final String decodedValue = (key.endsWith("Passwd") ? decodePassword(value) : value);
-            this.parameters.put(this.section + this.subsection + key, decodedValue);
+            if (assignmentOperatorPosition > 0) {
```

Comment:
why not use String::split()?

Reply:
More expensive. And uses a regex where no regex is needed.
No other particular reason.

Reply:
👍